src/idsk/components/header-web/template.njk

```diff
@@ -0,0 +1,315 @@
+                        }) }}
+                        <div class="idsk-header-web__main--login-action">
+                                <img class="idsk-header-web__main--login-action-profile-img" src="{{ main.loginButton.loggedIn.profileImg if main.loginButton.loggedIn.profileImg else '/public/assets/images/header-web/profile.svg' }}" alt="Profile image">
+                                <div class="idsk-header-web__main--login-action-text">
+                                        <span class="govuk-body-s idsk-header-web__main--login-action-text-user-name">{{ main.loginButton.loggedIn.userName if main.loginButton.loggedIn.userName else 'Ing. Jožko Veľký M.A' }}</span>
+                                        <div class="">
+                                                <a class="govuk-link idsk-header-web__main--login-action-text-logout" href="#" title="odhlásiť">{{ main.loginButton.logoutText if main.loginButton.logoutText else 'Odhlásiť' }}</a>
+                                                {% if main.loginButton.profileOption %} 
+                                                <span> | </span>
+                                                <a class="govuk-link idsk-header-web__main--login-action-text-profile" href="#" title="profil">{{ main.loginButton.profileOption if main.loginButton.profileOption != 'true' else 'Profil' }}</a>
+                                                {% endif %}     
+                                        </div>
+                                        
+                                </div>
+                        </div>
+                        
+                </div>
+        {% endif %}
+
+        {% if main.secondaryButton and params.headerType !== 'electronicService' %}
+                {{ idskButton({
+                        "text": main.secondaryButton.text if main.secondaryButton.text else 'Tlačidlo',
+                        "type": "button",
+                        "classes": main.secondaryButton.classes if main.secondaryButton.classes
+                }) }}
+        {% endif %}
+        </div>
+{% endset %}
+
+{% set searchInput %}
+        {% if main.search %}
+                <form class="idsk-header-web__main-action-search">
+                        <input class="govuk-input govuk-!-display-inline-block" id="idsk-header-web__main-action-search-input" name="search"
+                                placeholder="{{ params.search.label if params.search.label else 'Zadajte hľadaný výraz' }}" type="text" aria-describedby="input-width-30-hint" aria-label="Zadajte hľadaný výraz" />
+                        <button type="submit" class="govuk-button govuk-!-display-inline-block" data-module="govuk-button">
+                                <span class="govuk-visually-hidden">{{ main.search.button if main.search.button else "Vyhľadať" }}</span>
+                                <i aria-hidden="true" class="fas fa-search"></i>
+                        </button>
```

Comment:
Na ten search input mame component "search-component" a je tam aj ta ikona lupy, neviem ci nechceli tu nahodou iny ale mozno len do buducna aby si vedel ze staci pouzit macro, parametre a button je hotovy aj responzivny na mobil, aj ked sa bude menit styl toho search inputu s buttonom tak bude jednoduchsie zmenit jeden component ako vsade updatovat a stylovat.

Reply:
Ano suhlasim, dal by sa vyuzit, ale ja by som chcel casom ten search-component vymazat lebo nezapada to velmi do konceptu idsk podla mna. Cize za mna ok takto to moze ostat, jedine mozno tu ikonku reusenut ak je totozna. Pripadne ak mame nejake ikony ktore maju predpoklad byt na viacerych miestach nedavat to do folder-a /header-web/ ale vytvorme priecinok /icons/ aby sme eliminovali duplicity